sympy/integrals/transforms.py

```diff
@@ -1008,9 +1008,34 @@ def repl(ex, *args):
 def _laplace_transform(f, t, s_, simplify=True):
     """ The backend function for Laplace transforms. """
     from sympy import (re, Max, exp, pi, Min, periodic_argument as arg_,
-                       arg, cos, Wild, symbols, polar_lift)
+                       arg, cos, Wild, symbols, polar_lift, limit,
+                       Add, DiracDelta)
     s = Dummy('s')
-    F = integrate(exp(-s*t) * f, (t, 0, oo))
+    zero_minus = symbols('zero_minus', real=True)
+    # Expressions with DiracDelta need to be computed with a directional
+    # limit, but this will fail on some sums, so we first use the
+    # linearity of the Laplace transform to split the integral into a sum
+    # of integrals.
+    if f.has(DiracDelta):
+        f = f.expand(deep=True, modulus=None, power_base=False,
+                     power_exp=False, mul=True, log=False, multinomial=True,
+                     basic=True, complex=False, func=False, trig=False)
+        if f.func == Add:
```

Comment:
There is an attribute for efficiently testing this condition: `f.is_Add`.

Reply:
The things I learn here ... thank you! I will update right away, not least because the test pipeline has again failed due to a server problem. Is that normal on github that the tests run so unreliably?